Content.Server/GameTicking/Rules/PiratesRuleSystem.cs

```diff
@@ -235,6 +248,17 @@ public void MakePirate(Mind.Mind mind)
         if (!mind.OwnedEntity.HasValue)
             return;
         SetOutfitCommand.SetOutfit(mind.OwnedEntity.Value, "PirateGear", EntityManager);
+
+        // Notificate every player about a pirate antagonist role with sound
+        var pirateComponent = EntityManager.ComponentFactory.GetComponent<PiratesRuleComponent>();
```

Comment:
no need to use the factory here, take a look at how maketraitor works currently

Reply:
Made it like it's made in the traitor system, without deletion of a comment.